src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -2373,7 +2373,9 @@ private static bool FilterApplyPrefixLookup(MemberInfo memberInfo, string name,
         #region Private Data Members
 
 #pragma warning disable CA1823
+#pragma warning disable CS0169
```

Comment:
Is this needed? I do not see the change that would expose this warning.

Reply:
It kept failing for me locally. Let me try reverting the change though.

Reply:
Still repros for me locally.

Reply:
```
  LibraryImportGenerator -> E:\runtime\artifacts\bin\LibraryImportGenerator\Debug\netstandard2.0\Microsoft.Interop.LibraryImportGenerator.dll
  System.Private.CoreLib -> ...\runtime\artifacts\bin\System.Private.CoreLib\ref\Debug\net9.0\System.Private.CoreLib.dll
...\runtime\src\coreclr\System.Private.CoreLib\src\System\RuntimeType.CoreCLR.cs(2376,33): error CS0169: The field 'RuntimeType.m_keepalive' is never used [...\runtime\src\coreclr
\System.Private.CoreLib\System.Private.CoreLib.csproj]
```